sentry/src/main/java/io/sentry/Baggage.java

```diff
@@ -0,0 +1,187 @@
+package io.sentry;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@ApiStatus.Experimental
+public final class Baggage {
+
+  static final @NotNull String CHARSET = StandardCharsets.UTF_8.toString();
```

Comment:
@adinauer this is not supported on Android.
We have to `private static final Charset UTF_8 = Charset.forName("UTF-8");`

Reply:
We have to do https://github.com/getsentry/sentry-java/issues/1403 to avoid such issues, a simple and empty project already helps, set to API 16